### PR TITLE
[Stability checker] Provide summary for every test

### PR DESCRIPTION
**Pull request by aszecowka**
**Description**
Enhance stability checker to provider summary for every test.
An output of the test is examined against 2 regular expressions that indicate successful and failed test.

See also #484

### CODE REVIEW

File: tools/stability-checker/local/input/testing-dummy.sh [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+checkAndCleanupTest kyma-system
+checkErr=$?
+
+printImagesWithLatestTag
+latestTagsErr=$?
+
+
+
+if [ ${testErr} -ne 0] || [ ${latestTagsErr} -ne 0 ] || [ ${checkErr} -ne 0 ] ]
```
Comment (polskikiel) [inline tools/stability-checker/local/input/testing-dummy.sh]: Bad condition.
correct one:
`if [ ${testErr} -ne 0 ] || [ ${latestTagsErr} -ne 0 ] || [ ${checkErr} -ne 0 ]`

File: tools/stability-checker/README.md [hunk 2]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
 As another option, you can run the following:
 
 ```helm install https://github.com/kyma-project/stability-checker/raw/{branchName}/deploy/chart/stability-checker-0.1.0.tgz```
+
+## Local development
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]: `## Development`

- as in the readme template

File: tools/stability-checker/README.md [hunk 3]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
...
+For local development, following helpers can be used:
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]: `Use the following helpers for the local development:`

File: tools/stability-checker/README.md [hunk 4]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
...
+- `./local_minikube_build.sh` - builds Stability Checker docker image on minikube registry
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]:
```
- `./local_minikube_build.sh` which builds the Stability Checker Docker image on a Minikube registry.
- `./local/provision_volume.sh` which provisions a PersistentVolumeClaim (PVC) with testing scripts.
- `./local/charts/dummy` chart which contains simple and fast tests. To install it, execute the following command:
```

File: tools/stability-checker/README.md [hunk 5]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
...
+- `./local/provision_volume.sh` - provisions PVC with testing scripts.
+- chart `./local/charts/dummy` that contains simple and fast tests. To install it, execute following command:
+```
+helm install ./dummy --name dummy --namespace=kyma-system
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]: Try to use tab in here to make an indentation.

File: tools/stability-checker/README.md [hunk 6]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
...
+```
+- `./local_helm_install.sh` - install Stability Checker helm chart with predefined values. Note that, testing script points to `testing-dummy.sh`.
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]: ``- `./local_helm_install.sh` which installs the Stability Checker Helm chart with predefined values. The testing script points to the `testing-dummy.sh`.``

File: tools/stability-checker/README.md [hunk 7]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
...
+This script is a simplified version of `testing-kyma.sh`, where `dummy` chart is used to speed up testing.
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]: ``While this script is a simplified version of `testing-kyma.sh`, the `dummy` chart is used to speed up testing.``

File: tools/stability-checker/README.md [hunk 8]
```diff
@@ -26,3 +26,14 @@ Download the gzipped chart from:
...
+## Development
+Use the following helpers for the local development:
+- `./local_minikube_build.sh` which builds the Stability Checker Docker image on a Minikube registry.
+- `./local/provision_volume.sh` which provisions a PersistentVolumeClaim (PVC) with testing scripts.
+- `./local/charts/dummy` chart which contains simple and fast tests. To install it, execute the following command:
+```
+    helm install ./dummy --name dummy --namespace=kyma-system
+```
+- `./local_helm_install.sh` which installs the Stability Checker Helm chart with predefined values. 
+The testing script points to the `testing-dummy.sh`.This script is a simplified version of `testing-kyma.sh`, where `dummy` chart is used to speed up testing.
```
Comment (klaudiagrz) [inline tools/stability-checker/README.md]: ``The testing script points to the `testing-dummy.sh` which is a simplified version of `testing-kyma.sh`. The`dummy` chart is used in the `testing-dummy.sh` to speed up testing.``

**Pull request merged**